CHANGELOG.md

```diff
@@ -329,7 +329,7 @@ using this flag is insecure since verification is not carried out.
 
 - console: update graphiql explorer to support operation transform (#4567)
 - console: make GraphiQL Explorer taking the whole viewport (#4553)
-- console: fix table columns type comparision during column edit (close #4125) (#4393)
+- console: fix table columns type comparison during column edit (close #4125) (#4393)
```

Comment:
This is quite an outdated entry. Could you please revert this change?

Reply:
revert leaving wrong?

Reply:
CHANGELOG shouldn't be edited for anything else other than adding an entry for the current PR if required. Please remove the changes you have made here, that'll be great :)

Reply:
ok I did it, for your information on other repo I was able to modify changelog, and people suggested to add the new pr under improvement of current version. but you're the owner of your repo, so I will follow your suggestions :)